custom_dc/custom_data.md

```diff
@@ -17,14 +17,13 @@ Examples are provided in [`custom_dc/sample`](https://github.com/datacommonsorg/
 
 ## Prepare the CSV files {#prepare-csv}
 
-Custom Data Commons provides a simplified data model, which allows your data to be mapped to the Data Commons knowledge graph schema. Data in the CSV files should conform to a   
-_variable per column_ scheme. This requires minimal manual configuration; the Data Commons importer importer can create observations and statistical variables if they don't already exist, and it resolves all columns to [DCID](../glossary.md#dcid)s. 
+Custom Data Commons provides a simplified data model, which allows your data to be mapped to the Data Commons knowledge graph schema. Data in the CSV files should conform to a _variable per column_ scheme. This requires minimal manual configuration; the Data Commons importer importer can create observations and statistical variables if they don't already exist, and it resolves all columns to [DCID](../glossary.md#dcid)s. 
 
 With the variable-per-column scheme, data is provided in this format, in this exact sequence:
 
 _ENTITY, OBSERVATION_DATE, STATISTICAL_VARIABLE1, STATISTICAL_VARIABLE2, …_
 
-There is a single property, the _ENTITY_; all other properties must be expressed as [statistical variables](../glossary.md#variable). To illustrate what this means, consider this example: let's say you have a dataset that provides the number of public schools in U.S. cities, broken down by elementary, middle, secondary and postsecondary. Your data might have the following structure, which we identify as _variable per row_ (numbers are not real, but are just made up for the sake of example):
+There is a two properties, the _ENTITY_ and the _OBSERVATION_DATE_; all other properties must be expressed as [statistical variables](../glossary.md#variable). To illustrate what this means, consider this example: let's say you have a dataset that provides the number of public schools in U.S. cities, broken down by elementary, middle, secondary and postsecondary. Your data might have the following structure, which we identify as _variable per row_ (numbers are not real, but are just made up for the sake of example):
```

Comment:
nit: s/There is a/ There are

Reply:
another nit: It says "There are two properties... all other properties" but does not say what's special about those 2. Perhaps add something that talks about these 2 properties specifying the place and time of the observation?

Reply:
Aaargh, thanks for catching that one. And agree with the second one. PTAL.